project-management/zenhub.md

```diff
@@ -0,0 +1,83 @@
+# Working with ZenHub
+
+[ZenHub](https://zenhub.com/) is "agile project management within Github".
+We use it for all squads in the design team.
+
+Use [app.zenhub.com](https://app.zenhub.com), it's much nicer :).
+
+Current squad boards:
+
+- [Snappy squad tasks](https://app.zenhub.com/workspace/o/ubuntudesign/snappy-squad-tasks/boards)
+- [Web squad](https://app.zenhub.com/workspace/o/ubuntudesign/web-squad/boards)
+- [MAAS design squad](https://app.zenhub.com/workspace/o/ubuntudesign/maas-design-squad/boards)
+- [Juju squad](https://app.zenhub.com/workspace/o/ubuntudesign/juju-squad/boards)
+- [Vanilla squad](https://app.zenhub.com/workspace/o/ubuntudesign/vanilla-squad/boards)
+
+## 1. Iterations
+
+- Each iteration should be a milestone in ZenHub in the format `Iteration [Iteration number]-[enddate].[endmonth]`.
+
+ For example `Iteration 9-20.10` followed by `Iteration 10-3.11`.
+
+## 2. Epics
+
+- Each milestone (iteration) should include all epics to be worked on.
+- Epics should have everyone who will be working on it assigned.
+- Epics should always be in the `Iteration Epics` pipeline (see 4.3).
```

Comment:
I think maybe say "should" instead of "should always". I think "should always" should be written as "must" instead (to follow [RFC standards](https://tools.ietf.org/html/rfc2119)), but in this case I think it is sometimes valid to track the progress of the epic across the board, if the epic doesn't have sub tasks or those tasks exist off the current board.

Reply:
Agreed, will change

Reply:
changed